### DISCUSSION TITLE
Monographer Workflow - documenting specimens used

### DISCUSSION THREAD

**Discussion by debpaul** [mjy/monographers_do#4] [Q&A]
Several questions about **documenting specimens used** for a monograph.

1. Does someone who writes a monograph sometimes collect specimens they are / will refer to in the monograph? (I assume yes, at least sometimes).
2. IF the specimens are not yet deposited in a museum collection (i.e.  currently only in their personal collection), what recommendations would we make for the monographer to assign persistent identifiers for these specimens?
3. Upon depositing said specimens in a museum / university collection, what expectations do we have that said collection will preserve/publish these? What recommendations might we offer at this point? (We know that collections may assign their own identifiers for new specimens coming into the collection).

**Comment by mjy** [comment]
Here is my take, please be skeptical of it.

> Does someone who writes a monograph sometimes collect specimens they are / will refer to in the monograph? (I assume yes, at least sometimes).

Yes.  At least in Zoology I think you can assume most times, if not nearly all.

> IF the specimens are not yet deposited in a museum collection (i.e. currently only in their personal collection), what recommendations would we make for the monographer to assign persistent identifiers for these specimens?

Do you mean digital or physical?

If physical (the most important of the two IMO) assign a paper label catalog number (using archival paper/ink/processes etc.)  These typically have 2 components, a "Namespace", and a variable bit.  Do some research to try and make the Namespace unique.  Don't sweat it though, this is just one identifier that will "localize" people to the specimens you looked at (see below).  Perhaps also integrate a QR or 128 version of the identifier on the physical label (software like TW or spreadsheets with a formula can help do this). Perhaps, if the specimen is large enough, use a [UUID](https://en.wikipedia.org/wiki/Universally_unique_identifier) as the variable bit.  Why is the paper the most important?  It eliminates the most crucial step in future research- confirming you're looking at the same specimen as your precursors when that specimen is in front of you. Again, this is just one identifier for your specimen, curratorial platforms will add more, and digital identifiers typically a(nother) UUID and staging id (e.g. internal database id).

_Remember, identifiers "localize" (get you in the vicinity of data) their job is to get you close enough that your effort is minimized in confirming that you have the data in hand so that you can do your next bit of work. I would argue this holds for both physical and digital forms, others would disagree particularly on the latter._

> Upon depositing said specimens in a museum / university collection, what expectations do we have that said collection will preserve/publish these? What recommendations might we offer at this point? (We know that collections may assign their own identifiers for new specimens coming into the collection).

A ridiculously high expectation, because one assumes they will also be "published".  Any collection worth its while will recognize the importance of these identifiers, i.e. they should be doing the same before the loans go out.  If the collection adds another (don't remove the first) of their own catalog number as well, no-big deal, modern systems allow for multiple identifiers on specimens.

The only we should infer from identifiers is identity, in this case of a physical specimen, or the metadata around it. Identifiers should never assert/do anything more (e.g. ownership, sets, resolving, etc., those types of things are the responsibility of other processes, tools, metadata etc.

**Comment by nancybsimmons** [comment]
From a practical perspective, my experience with collections (and collections managers) suggests that while they *may* be willing to add a UUID/GUID to a specimen, many simply never will.  Accessioning and cataloging with the legacy catalog numbering system of the museum or repository will always be their first priority.  It's always going to be up the systematist/monographer to get persistent identifiers assigned if they want them associated with the specimens they're using.

**Comment by mjy** [comment]
Identifiers is a "black hole" topic.  Monographers are practical people, they have data to compile, specimens to look at, loans to request, images to make, trips to take, teams to assemble!  If we focus on their practical needs, then we can hand off work to others in a "fair" way.  For example, let's phrase the issue this way:

* I need to answer the question "Who looked at this specimen?" (Q - question) to the next taxonomists looking at this specimen (A - audience) with the data "some identifier" (D - data) in the format (F -format) of a physical code attached near the physical specimen.

Given this practical need, we can look at the two types of identifiers being discussed.
1) "Global" E.g. a UUID like `b25646da-8764-4090-aa93-487eb602dbb8`
2) "Local" E.g. a Namespace + alphanumeric like `INHS Insect Collection 1231828`

In this case, a printed version of either string, `b25646da-8764-4090-aa93-487eb602dbb8` or `INHS Insect Collection 1231828` physically attached to the specimen answers the question (where "answers" is, it's good enough).  As Monographers, we're good with this, we see a reference to the identifiers somewhere, compare it to the thing we are looking at, and we move on.

So what's the problem?  The techy-people (and others, including monographers, curators, scientists, etc.) have some other question(s) that they want monographers to answer as part of the monographic process (which, remember is the focus here, as opposed to solving the worlds biodiversity informatics problems), however what this problem is, without a precise definition like above, is vague and frustrating to deal with.

If questions can be formatted  in the format above, then maybe the monographers and those supporting them will have a fighting chance of meeting their practical needs.  Let's look at a second question:

* To answer the question "Why didn't you attach an identifier to a specimen you saw, but could not touch (or did not have permission to touch)?" (Q - question) to the editor of a journal/biodiversity informatician/fellow monographer (A - Audience) I need documentation of my frustrating communications with the institute responsible for not putting the identifier on my specimen by my request (D - Data) in the format of a PDF (F - Format).  

How does this let the monographer move on?  We, the monographers, hand off the PDF, "evidence of non-compliance" to the admins/editors, and they do their social pressuring.  I.e. it shouldn't be a monographers job (alone) to pressure institutes or organizations to follow best practices, this is something that needs to happen at the community level.

This is obviously a little ranty, but it highlights a big frustration of mine with engaging these types of questions that assume certain things without context of the question that is being tackled. "Identifiers are important!  Catalog numbers are not good enough!  UUIDs are the answer!"  Yes, and no.
